gratipay/security/authentication.py

```diff
@@ -82,9 +81,7 @@ def add_auth_to_response(response, request=None, user=ANON):
     if request is None:
         return  # early parsing must've failed
     if request.line.uri.startswith('/assets/'):
-        return  # assets never get auth headers and have their own caching done elsewhere
-
-    response.headers['Expires'] = BEGINNING_OF_EPOCH # don't cache
```

Comment:
Yes, we don't need `Expires` here, and we don't want it here either: a function that handles auth shouldn't set caching headers, it's not its job.